RunningOrder/Shared/Extensions/Combine+Extensions.swift

```diff
@@ -106,8 +108,8 @@ extension Publisher where Self.Output == Never {
     }
 
     /// A combine sink version to ignore receiveValue completion block when Self.Output == Never
-    /// - Parameter completion: <#completion description#>
-    /// - Returns: <#description#>
+    /// - Parameter completion: completion description
+    /// - Returns: description
```

Comment:
the documentation doesn't look correct

Reply:
true, I just removed placeholder style 😅